src/util/routerinfo.cc

```diff
@@ -31,10 +31,13 @@
 #include "util/routerinfo.h"
 #include <assert.h>
 #include <memory>
+
+#include "core/crypto/rand.h"
 #include "core/router/info.h"
 #include "core/util/exception.h"
 #include "core/util/filesystem.h"
 #include "core/util/log.h"
+#include "version.h"  // NOLINT(build/include)
 
 namespace bpo = boost::program_options;
 
```

Comment:
Will you consider adding `namespace core = kovri::core` and a subsequent refactor in order shorten access (as described in our upstream style guide)?